src/sentry/tasks/check_monitors.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import absolute_import, print_function
+
+import logging
+
+from django.utils import timezone
+
+from sentry.models import Monitor, MonitorStatus, MonitorType
+from sentry.tasks.base import instrumented_task
+
+
+logger = logging.getLogger('sentry')
+
+
+@instrumented_task(name='sentry.tasks.check_monitors', time_limit=15, soft_time_limit=10)
+def check_monitors():
```

Comment:
How does this get called?

Reply:
doesnt yet per the diff, but it will be a cron job